samples/snippets/src/main/java/com/example/monitoring/ListTimeSeries.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.monitoring;
+
+// [START monitoring_read_time_series_simple]
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.monitoring.v3.ListTimeSeriesRequest;
+import com.google.monitoring.v3.ProjectName;
+import com.google.monitoring.v3.TimeInterval;
+import com.google.protobuf.util.Timestamps;
+import java.io.IOException;
+
+// Sample to list time series
+public class ListTimeSeries {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String filter = "your-metric-type-filter";
+    listTimeSeries(filter, projectId);
+  }
+
+  public static void listTimeSeries(String filter, String projectId) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (MetricServiceClient metricServiceClient = MetricServiceClient.create()) {
+      ProjectName projectName = ProjectName.of(projectId);
+
+      // Restrict time to last 20 minutes
+      long startMillis = System.currentTimeMillis() - ((60 * 20) * 1000);
+      TimeInterval interval =
+          TimeInterval.newBuilder()
+              .setStartTime(Timestamps.fromMillis(startMillis))
+              .setEndTime(Timestamps.fromMillis(System.currentTimeMillis()))
+              .build();
+
+      // Prepares the list time series request
+      ListTimeSeriesRequest request =
+          ListTimeSeriesRequest.newBuilder()
+              .setName(projectName.toString())
+              .setFilter(filter)
+              .setInterval(interval)
+              .build();
+
+      // Send the request to list the time series
+      MetricServiceClient.ListTimeSeriesPagedResponse response =
+          metricServiceClient.listTimeSeries(request);
+
+      // Process the response
+      System.out.println("Got timeseries: ");
+      response.iterateAll().forEach(timeSeries -> System.out.println(timeSeries));
+    } catch (ApiException ex) {
+      System.out.print("\n time series not found." + ex.toString());
+    }
```

Comment:
\n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

Reply:
Done